docs/src/prototypes/chatMessages/ControlMessages/GroupControlMessages.tsx

```diff
@@ -0,0 +1,107 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import * as keyboardKey from 'keyboard-key'
+
+import { List, chatBehavior, Accessibility, Flex, Icon } from '@stardust-ui/react'
+import ControlMessage from './ControlMessage'
+
+const controlMessagesGroupBehavior: Accessibility<any> = props => {
+  const behaviorData = chatBehavior(props)
+
+  behaviorData.attributes.root = {
+    ...behaviorData.attributes.root,
+    'data-is-focusable': true,
+  }
+
+  behaviorData.focusZone = {
```

Comment:
maybe it would be more readable if we replace only the props here, not the whole `focusZone` object.